### PR TITLE
Remove print statements and improve logging

### PR DESCRIPTION
**Pull request by J535D165**
This PR removes print statements in the REST api and SQlock handler. Print statements are replaced by logging statements that start with the project identifier. 

@qubixes can you have a look at the changes in the sqlock file?

### CODE REVIEW

File: asreview/webapp/api.py [hunk 1]
```diff
@@ -389,11 +389,11 @@ def api_get_project_data(project_id):  # noqa: F401
         statistics["filename"] = filename
 
     except FileNotFoundError as err:
-        print(err)
+        logging.info(err)
```
Comment (qubixes) [inline asreview/webapp/api.py:392]: Should be logging.error probably?

**Comment by qubixes** [comment]
@J535D165 It doesn't seem like anything material changed except the inclusion of the project_id + logging? Seems all good to me.

File: asreview/webapp/api.py (hunk 1 above)
Comment (J535D165) [reply asreview/webapp/api.py:392]: Thanks. This one is oke. No file found is a 200 response.

**Comment by J535D165** [comment]
Yes, correct. No big changes. 

I'm facing some issues on displaying the logging messages in the stdout right now. Not completely sure how to deal with this. I would like to have all logging (asreview core, asreview lock manager, asreview REST API) statements in my stdout. Any ideas?

**Comment by qubixes** [comment]
I'm sorry, I have no idea about everything in stdout. The problem is that some of it goes to stderr? Maybe it can be redirected on the command line?

**Comment by J535D165** [comment]
My bad, this had to do with the log level and some other things. It's more or less resolved now. Thanks.

**Pull request merged**